packages/docusaurus-1.x/lib/core/Head.js

```diff
@@ -172,6 +172,35 @@ class Head extends React.Component {
             }}
           />
         )}
+        <script src="https://unpkg.com/smooth-scroll@16.0.3/dist/smooth-scroll.polyfills.min.js" />
+        <script src="/js/scrollSpy.js" />
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `
+              document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Use the ES5 function syntax because these JS scripts/code will not be transpiled at all.